### PR TITLE
add dag ctr

### PR DESCRIPTION
**Pull request by cooper-lzy**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: docs-2.0/graph-computing/0.deploy-controller-analytics.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+# Dag Controller
+
+Dag Controller is a task scheduling tool that can schedule the jobs which type is DAG (directed acyclic graph). The job consists of multiple tasks to form a directed acyclic graph, and there is dependency between the tasks.
```
Comment (foesa-yang) [inline docs-2.0/graph-computing/0.deploy-controller-analytics.md:3]:
```suggestion
Dag Controller is a task scheduling tool that can schedule the jobs which type is DAG (directed acyclic graph). The job consists of multiple tasks to form a directed acyclic graph, and there is a dependency between the tasks.
```

File: docs-2.0/graph-computing/0.deploy-controller-analytics.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+## Prerequisites
+
+- The [HDFS](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/ClusterSetup.html) 2.2.x or later has been deployed.
+
+- The JDK 1.8 has been deployed.
+
+## Preparations
+
+There are some differences between installation packages and commands in different environments. The preparations is as follows
```
Comment (foesa-yang) [inline docs-2.0/graph-computing/0.deploy-controller-analytics.md:21]:
```suggestion
There are some differences between installation packages and commands in different environments. The preparations are as follows
```

File: docs-2.0/graph-computing/0.deploy-controller-analytics.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+- The operating system is CentOS 7.
+- If the Nebula Analytics and the Dag Controller are deployed on multiple machines, ensure network connectivity between the machines.
+- If the Nebula Analytics is a cluster with distributed architecture, ensure the paths and ports are configured identically for each machine.
+
+## Precautions
+
+- The BFS and SSSP algorithms need to verify the parameter `root`. They support only one upstream component and must specify rows and columns. If multiple files exist, a random file is selected. If row, column, or file is not found, an error will be reported.
```
Comment (foesa-yang) [inline docs-2.0/graph-computing/0.deploy-controller-analytics.md:29]:
```suggestion
- The BFS and SSSP algorithms need to verify the parameter `root`. They support only one upstream component and must specify rows and columns. If multiple files exist, a random file is selected. If a row, column, or file is not found, an error will be reported.
```

File: docs-2.0/graph-computing/0.deploy-controller-analytics.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+Reconfigure the permissions to `744` on the folder `.ssh` and `600` on the file `.ssh/authorized_keys`.
+
+### How to resolve the error `There are 0 Nebula Analytics available. clusterSize should be less than or equal to it`?
+
+The possible causes are as follows:
+
+- The Nebula Analytics has not been deployed. Configure the Nebula Analytics as described in this document.
+
+- The Nebula Analytics has been deployed, but can not connect to the Dag Controller. For example, the IP address is incorrect, SSH is not configured, and the startup users of the two service are inconsistent (causing SSH login failures).
```
Comment (foesa-yang) [inline docs-2.0/graph-computing/0.deploy-controller-analytics.md:180]:
```suggestion
- The Nebula Analytics has been deployed, but can not connect to the Dag Controller. For example, the IP address is incorrect, SSH is not configured, and the startup users of the two services are inconsistent (causing SSH login failures).
```

File: docs-2.0/graph-computing/0.deploy-controller-analytics.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
+# Dag Controller
+
+Dag Controller is a task scheduling tool that can schedule the jobs which type is DAG (directed acyclic graph). The job consists of multiple tasks to form a directed acyclic graph, and there is a dependency between the tasks.
+
+The Dag Controller can perform complex graph computing with the Nebula Analytics. For example, the Dag Controller sends an algorithm request to the Nebula Analytics, which saves the result to Nebula Graph or HDFS. The Dag Controller then takes the result as input to the next algorithmic task to create a new task.
```
Comment (foesa-yang) [inline docs-2.0/graph-computing/0.deploy-controller-analytics.md:5]: Maybe you can link to Nebula Analytics?

**Review by abby-cyber — APPROVED** [review]

File: docs-2.0/graph-computing/0.deploy-controller-analytics.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+  ```
+  export HADOOP_HOME=<hadoop_path>
+  export JAVA_HOME=<java_path>
+  ```
+
+## Deploy Dag Controller
+
+1. Complete the SSH password-free configurations so that the Dag Controller machine can log directly into the Nebula Analytics machines and all machines within the Nebula Analytics cluster can connect directly to each other without passwords.
+
+  For example, the user in the machine A (Dag Controller) log directly into machine B-1 in the Nebula Analytics cluster over SSH  without passwords. Run the following commands on the machine A:
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/0.deploy-controller-analytics.md:66]:
```suggestion
  For example, the user in the machine A (Dag Controller) log directly into machine B-1 in the Nebula Analytics cluster over SSH without passwords. Run the following commands on the machine A:
```

**Review by whitewum — APPROVED** [review]

**Pull request merged**